tests/functional/p/possible_forgotten_f_prefix.py

```diff
@@ -0,0 +1,53 @@
+"""Check various forms of strings which could be f-strings without a prefix"""
+# pylint: disable=invalid-name, line-too-long, pointless-string-statement, pointless-statement
+# pylint: disable=missing-function-docstring
+
+# Check for local variable interpolation
+PARAM = "string"
+PARAM_TWO = "extra string"
+
+A = f"This is a {PARAM} which should be a f-string"
+B = "This is a {PARAM} used twice, see {PARAM}"
```

Comment:
```suggestion
B = "This is a {PARAM} used twice, see {PARAM}"  # [possible-forgotten-f-prefix, possible-forgotten-f-prefix]
```

Reply:
I have intentionally left this as a case without warnings. Whenever users use the same `parameter` twice it is likely that they are going to use `format()` later on.
See lines 950-952:
```python
        # If a variable is used twice it is probably used for formatting later on
        if len(inner_matches) != len(set(inner_matches)):
            return
```